runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/state/StateRequestHandlers.java

```diff
@@ -414,6 +432,21 @@ public static StateRequestHandler forBagUserStateHandlerFactory(
       }
     }
 
+    @Override
+    public Iterable<BeamFnApi.ProcessBundleRequest.CacheToken> getCacheTokens() {
+      return handlerCache.values().stream()
```

Comment:
nit: [streams are siginficantly slower than for loops even with JDK 12](https://medium.com/@milan.mimica/slow-like-a-stream-fast-like-a-loop-524f70391182)

Reply:
Not sure how much this contributes here to the overall computing, but the performance of the blog post indeed looks pretty bad. I've replaced this with for loops.

Reply:
It grinds my gears when people use streams for things because they want a fluent builder pattern for processing when a for loop is easier to read/understand for the majority of people.

Reply:
It depends what you are used to. I find the fluent pattern easier to read, but I agree that the procedural definition is probably easier for most people.